src/client/index.ts

```diff
@@ -856,8 +856,8 @@ export class Client {
 	async eventSender(eventName: 'tokens', data: EventSenderTokens)
 	async eventSender(eventName: 'token-proof', data: EventSenderTokenProof)
 	async eventSender(eventName: 'tokens-selected', data: EventSenderTokensSelected)
-	async eventSender(eventName: 'connected-wallet', data: EventSenderConnectedWallet)
-	async eventSender(eventName: 'disconnected-wallet', data: EventSenderDisconnectedWallet)
+	async eventSender(eventName: 'connected-wallet', EventSenderConnectedWallet)
```

Comment:
This is incorrect, as type is specified as a parameter name. 

To fix:
1. Revert these lines
2. Change EventSenderConnectedWallet to a named union type rather than an interface: 
`export type EventSenderConnectedWallet = WalletConnection | null`

Reply:
Thanks @micwallace. As confirmed via Discord, please could you help address this interface/type issue raised.
Adding directly to this PR is fine / most efficient for us addressing this.